index.html

```diff
@@ -233,9 +233,9 @@ <h2>Languajes
         <h1>Contact me</h1>
         <p>If you have an application you are interested in developing, a feature 
             that you need built or a project that needs coding. I’d love to help with it </p>
-            <input type="text" id="form-name" placeholder="Yeremias">
-            <input type="email" id="form-email" placeholder="natajayanj14@gmail.com">
-            <input type="text" id="form-message" placeholder="Write your message here">
+            <input type="text" id="form-name" placeholder="Enter name" maxlength="30">
+            <input type="email" id="form-email" placeholder="Enter email">
+            <input type="text" id="form-message" placeholder="Write your message here" maxlength="500">
```

Comment:
- [ ] As requested by the previous reviewer, kindly make use of the `form` tag in creating your [form.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form)
- [ ] Also, your message area although it is an input field, it is required you to use a special input tag called `textarea` and should have `maxlength="500"`. This gives the user a much bigger space to write in. You can have a look at this [resource](https://www.w3schools.com/tags/tag_textarea.asp) on how to do that.
- [ ] It is also a requirement that you make the form inputs required by adding form validation `required` to each form input.